.pipelines/diabetes_regression-variables-template.yml

```diff
@@ -63,12 +63,6 @@ variables:
   # - name: ALLOW_RUN_CANCEL
   #   value: "true"
 
-    # For debugging deployment issues. Specify a build id with the MODEL_BUILD_ID pipeline variable at queue time
-    # to skip training and deploy a model registered by a previous build.
-  - name: modelbuildid
-    value: $[coalesce(variables['MODEL_BUILD_ID'], variables['Build.BuildId'])]
```

Comment:
you should remove this from here too https://github.com/microsoft/MLOpsPython/blob/1d2d7733608b010610c98043d0d2a8a0e5e3b5bb/.pipelines/diabetes_regression-variables-template.yml

Reply:
Looks like that's the same file? Let me know if I'm mistaken